packages/cron/src/bin/names.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+import fetch from '@web-std/fetch'
+import { postNamesToW3name } from '../jobs/names.js'
+import { envConfig } from '../lib/env.js'
+import { getCluster, getDBClient } from '../lib/utils.js'
+
+/** @ts-ignore */
+global.fetch = fetch
+
+async function main () {
+  const env = process.env.ENV || 'dev'
+  const db = getDBClient(process.env)
+
+  await postNamesToW3name({ env, db })
+}
+
+envConfig()
+
+main()
```

Comment:
What you've got here is the same as the other cron jobs we've got, but should we be doing `await main()` here instead?

Reply:
Good point, but if I add `await`, it complains about having an async call inside a non-async context. Other cron jobs don't use it too.

Reply:
Ah yeah, sorry, my bad. I'd forgotten that it doesn't apply on the outermost function 🤦 .